Assets/HoloToolkit/Input/Scripts/Voice/KeywordManager.cs

```diff
@@ -19,17 +21,15 @@ namespace HoloToolkit.Unity.InputModule
     /// Edit -> Project Settings -> Player -> Settings for Windows Store -> Publishing Settings -> Capabilities
     /// or in your Visual Studio Package.appxmanifest capabilities.
     /// </summary>
-    public partial class KeywordManager : MonoBehaviour
+    public partial class KeywordManager : BaseInputSource
     {
         [System.Serializable]
-        public struct KeywordAndResponse
+        public struct KeywordAndKeyCode
         {
             [Tooltip("The keyword to recognize.")]
             public string Keyword;
             [Tooltip("The KeyCode to recognize.")]
             public KeyCode KeyCode;
-            [Tooltip("The UnityEvent to be invoked when the keyword is recognized.")]
-            public UnityEvent Response;
```

Comment:
The propose of this PR is to use the InputManager handling mechanisms so I thought the UnityEvent wouldn't make sense anymore.
Anyway, I'm going to follow the suggestion of creating a new SpeechInputSource and leave the KeywordManager like it was.